src/main/kotlin/karazin/scala/users/group/week1/homework/program.kt

```diff
@@ -2,10 +2,34 @@ package karazin.scala.users.group.week1.homework
 
 import karazin.scala.users.group.week1.homework.adt.ErrorOr
 import karazin.scala.users.group.week1.homework.model.PostView
+import karazin.scala.users.group.week1.homework.services.getComments
+import karazin.scala.users.group.week1.homework.services.getPosts
+import karazin.scala.users.group.week1.homework.services.getUserProfile
+import karazin.scala.users.group.week1.homework.services.getLikes
+import karazin.scala.users.group.week1.homework.services.getShares
 
 object program {
 
-    fun getPostsViews(): ErrorOr<List<ErrorOr<PostView>>> = throw NotImplementedError()
-    fun getPostView(post: model.Post): ErrorOr<PostView> = throw NotImplementedError()
+    fun getPostsViews(): ErrorOr<List<ErrorOr<PostView>>> =
```

Comment:
And now please changed the result type to `ErrorOr<List<PostView>>`. Update the implementation of the function accordingly.